variants/sea_picro/pins_arduino.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+// Pin definitions taken from:
+//    https://github.com/joshajohnson/sea-picro/blob/master/documentation/pinout/sea-picro-top-pinout-rory-render.png
+
+// LEDs not pinned out
+#define PIN_LED (31u)
```

Comment:
While per schematics the board doesn't have a simple on/off LED, it does have a Neopixel on GPIO25. Hence you should add a `PIN_NEOPIXEL` definition like other boards do (like a lot from Adafruit)

https://github.com/earlephilhower/arduino-pico/blob/f5f7267f4459d83652ffdec8d3acf4fb9fe743b1/variants/adafruit_metro/pins_arduino.h#L6-L7

Reply:
Even if pin 25 is not actually connected to a NeoPixel? It's meant for driving them, but there is no actual LED on the board.

Reply:
Ah on the "1.0-EXT" variant that's just connected to the `74LVC1T45` driver / level-shifter chip, but not to an actual LED.

But on the "1.0-RST" variant it does show an actual WS2812C-2020 as D3. But this PR is for the "1.0-EXT" variant?

Reply:
Correct, for the EXT. Should I indicate that somehow? Preferred formatting?

Reply:
If it's a really different board variant I would have created `variants/sea_picro_1_0_ext` and `variants/sea_picro_1_0_rst`, but if it's about the Neopixel driver vs actual Neopixel and they're even on the same pin, I think this is fine too.

Reply:
Yeah, the only other difference is in the reset circuit for the RP2040, so I don't think it makes much sense to maintain two copies of everything unless you feel differently.